src/rprblender/nodes/blender_nodes.py

```diff
@@ -1901,6 +1901,57 @@ def export_hybrid(self):
         return None
 
 
+class ShaderNodeTexVoronoi(NodeParser):
+    """Create RPR Voronoi node"""
+
+    def export_rpr2(self):
+        if self.node.voronoi_dimensions in ('1D', '4D'):
+            log.warn("Unsupported dimension type", self.node.voronoi_dimensions, self.node,
```

Comment:
I've set warns for '1D' and '4D' because of unsupported.
pyrpr.MATERIAL_INPUT_DIMENSION uses int indexies
1 = '1D' (unsupported warn)
2 = '2D' in practice looks like 3D
3 = '3D' looks correct
4 = '4D' (unsupported warn)
![photo_2021-06-29_20-40-38](https://user-images.githubusercontent.com/42581166/123843803-f6ec7200-d91a-11eb-9b7a-daba9934e3e2.jpg)

Reply:
is 2D wrong or just looks like 3D?

Reply:
Looks like 2D RPR is wrong. Result is different compare to 2D Blender node.
pyrpr.MATERIAL_INPUT_DIMENSION = 2, 3 produces the same result.

Reply:
Should support be left for 3D only?